### ISSUE TITLE
Results on Pascal VOC

### ISSUE THREAD

**Issue by Srinivasa-Divakar-Bhat** [LiheYoung/UniMatch#60]
Hi,
I am running the code on Pascal VOC for the following setting. I am using the data provided in the repo.
But the accuracy I am able to achieve is 77.19 which is apparently closer to the result in Table 3 in the Paper ( which is corresponding to the case when Labeled images are sampled from the blended training set). Is there anything I am missing? Or is  there any specific configuration setting that I can use to generate the results for the  original high-quality training set.
Thanks in advance. 

[2023-07-12 01:24:10,246][    INFO] {'backbone': 'resnet101',
 'batch_size': 2,
 'conf_thresh': 0.95,
 'config': 'configs/pascal.yaml',
 'criterion': {'kwargs': {'ignore_index': 255}, 'name': 'CELoss'},
 'crop_size': 321,
 'data_root': '/path/to/data',
 'dataset': 'pascal',
 'dilations': [6, 12, 18],
 'epochs': 80,
 'labeled_id_path': 'splits/pascal/1_4/labeled.txt',
 'local_rank': 0,
 'lr': 0.001,
 'lr_multi': 10.0,
 'model': 'deeplabv3plus',
 'nclass': 21,
 'ngpus': 4,
 'port': 20017,
 'replace_stride_with_dilation': [False, False, True],
 'save_path': 'exp/pascal/unimatch/base_1/1_4',
 'unlabeled_id_path': 'splits/pascal/1_4/unlabeled.txt',}

**Comment by LiheYoung** [comment]
Your currently used split ``splits/pascal/1_4/labeled.txt`` is just for Table 3, just as you mentioned.

If you want to reproduce the results of sampling labeled images from the high-quality training set (Table 2), you should use splits ``splits/pascal/$num/labeled.txt``, where ``num`` can be 92, 183, 366, 732, and 1464.

**Comment by Srinivasa-Divakar-Bhat** [comment]
Thank you for the response.

**Comment by zhk779** [comment]
> Thank you for the response.
Hi, Have you managed to reproduce the precision on the Pascal VOC dataset in the end?
